tsdb/head.go

```diff
@@ -840,24 +863,33 @@ func (a *initAppender) Rollback() error {
 func (h *Head) Appender() storage.Appender {
 	h.metrics.activeAppenders.Inc()
 
+	writeID := h.iso.newWriteID()
```

Comment:
While the writeID is created during appender creation, I think it would be confusing to call it appenderID. This is propagated everywhere, including chunk iterators. When reading from the TSDB, you probably don't care if writes are done by appenders or batchers or scrapers.

writeID is also the term used in Brian's talk. I think the terminology makes sense.

Reply:
As per discussion on IRC, how about `appendID`? It would convey that it is per appender, while it would still somewhat keep the symmetry between _read_ and _append_ (formerly _write_).

Reply:
:+1:

Reply:
Done.